### PR TITLE
Add prometheus component

### PR DESCRIPTION
**Pull request by rcloran**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4444

### CODE REVIEW

File: source/_components/prometheus.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: "Prometheus"
+description: "Record events in Prometheus."
+date: 2016-11-16 08:00
+sidebar: true
+comments: false
+sharing: true
+logo: prometheus.png
+footer: true
+ha_category: "History"
+ha_release: 0.12
```
Comment (Landrash) [inline source/_components/prometheus.markdown]: Wrong Home Assistant release number. Will most likely be 0.34 or greater.

**Comment by fabaff** [comment]
This main PR seems to have gone stale. Closing this as well. You can reopen it when you're ready to finish it.

**Comment by raccettura** [comment]
Any chance on this PR being picked up? It would be great to capture the data HA is getting from various sensors into a way that they can be analyzed (shiny graphs).

**Comment by GussyH** [comment]
I'd like this too - need to get my head around Prometheus for work and using my HASS install would be a good excuse :)

**Comment by GussyH** [comment]
@rcloran is this still in development?

**Comment by blackbarn** [comment]
quite unfortunate this isn't a thing yet. It would be very useful!

**Comment by rcloran** [comment]
Updated with new PR.

**Comment by rcloran** [comment]
Reopen for reals.

**Comment by rcloran** [comment]
Nope, github UI fail. #2913

**Pull request closed without merging**